windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
+
+MAINTAINER Azavea
+
+RUN apt-get update
+
+# Windshaft requires node < 11; vivid has 10.25 in nodejs package
+RUN apt-get -q install -y nodejs npm git-core \
+    libcairo2-dev libpango1.0-dev libjpeg8-dev libgif-dev libpq-dev
+
+# link nodejs command to node
+RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
+
+RUN mkdir -p /opt/windshaft
+WORKDIR /opt/windshaft
+COPY . /opt/windshaft
+
+# NB: the most recent versions do not work with the provided configurations
+RUN npm install windshaft@0.36.0
+
+EXPOSE 5000
+
+CMD ["node", "server.js"]
```

Comment:
Not clear whether we expect to ever run anything besides `node server.js`, but if we do (maybe tests) we can make either `node` or `npm` (and use the `scripts` option in `package.json`) the `ENTRYPOINT`. That makes it so that arguments at the end of `docker run` get used as arguments for the `ENTRYPOINT` command.

When `ENTRYPOINT` exists, `CMD` becomes the arguments to `ENTRYPOINT`. So to keep the current approach as the default, it could be:

```dockerfile
ENTRYPOINT ["/usr/bin/node"]
CMD ["server.js"]
```

A potentially clearer example of this approach is [here](https://github.com/azavea/docker-aws-cli).